python-sdk/tests/files/type/test_base.py

```diff
@@ -16,12 +20,14 @@ def test_repr():
     )
 
 
-def test_eq():
+@patch("astro.files.locations.base.BaseFileLocation.validate_conn", return_value=None)
```

Comment:
shouldn't be needed

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1478/commits/b89f84be043188126f642e50de139bf330adb72b)